CHANGELOG.md

```diff
@@ -3,52 +3,58 @@
 FEATURES:
 
 * **New Resource:** `vcd_vm_internal_disk` VM internal disk configuration [GH-412]
-* `vcd_vapp_vm` Internal disks in VM template allows to be edited by `override_template_disk` field
-  [GH-412]
-* `vcd_org_vdc` can be created with Flex allocation in vCD 9.7 and later. Also two new fields added for Flex - `elasticity`, `include_vm_memory_overhead` [GH-443]
-* `resource/vcd_vapp_vm` and `datasource/vcd_vapp_vm` get optional `network_dhcp_wait_seconds` field
-  to ensure `ip` is reported when `ip_allocation_mode=DHCP` is used [GH-436]
 * **New Resource:** `vcd_vapp_org_network` vApp organization network [GH-455]
-* `vcd_vapp_network` supports isolated network and vApp network connected to Org VDC networks [GH-455]
 * **New Data Source:** `vcd_vapp_org_network` vApp org network [GH-455]
 * **New Data Source:** `vcd_vapp_network` vApp network [GH-455]
-* `resource/vcd_vapp_vm` and `datasource/vcd_vapp_vm` `customization` block supports all available features [GH-462]
 
 IMPROVEMENTS:
 
+* `vcd_vapp_network` supports isolated network and vApp network connected to Org VDC networks
+  [GH-455,GH-468]
+* `vcd_vapp_network` does not default `dns1` and `dns2` fields to 8.8.8.8 and 8.8.4.4 respectively
```

Comment:
For me it feels more a bugfix than improvement. @vbauzysvmware ?

Reply:
It wasn't bug, we decided it will work better without default. We had such request in another resource.